src/lib.rs

```diff
@@ -253,6 +299,20 @@ fn open_browser_internal(browser: Browser, url: &str) -> Result<ExitStatus> {
     }
 }
 
+#[cfg(any(
+    target_os = "linux",
+    target_os = "freebsd",
+    target_os = "netbsd",
+    target_os = "openbsd"
+))]
+fn adapt_command(cmd: &mut Command, suppress_output: bool) -> &mut Command {
```

Comment:
Please allow this to accept `BrowserOptions` directly, as I feel it'd be best for later behavioural modifications as well.